desktop.css

```diff
@@ -317,4 +833,154 @@
   form > * {
     margin-bottom: 10px;
   }
+
+  /* PopUp Window */
+
+  .popup {
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    min-height: 100%;
+    z-index: 1000;
+    overflow-y: scroll;
+  }
+
+  .popup.active {
+    display: inline-block;
+
+    /*  filter: blur(4px); */
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments.